packages/core/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "aws-xray-sdk-core",
-  "version": "3.5.0",
+  "version": "3.4.1",
```

Comment:
This corresponds to the [current release version (3.5.0)](https://github.com/aws/aws-xray-sdk-node/releases/tag/aws-xray-sdk-node%403.5.0) and should never be changed to a prior version - this version number is only modified by the X-Ray team when preparing a new release. I imagine this is what is causing most of the CI failures. 

To fix this, please keep this version as is - 3.5.0. We have an [sdk_contrib directory](https://github.com/aws/aws-xray-sdk-node/tree/master/sdk_contrib) that houses all community contributed instrumentations, so it would be great if this instrumentation could be moved there. 

Also thought it would be worth mentioning since we do not usually maintain instrumentations in sdk_contrib, would you be willing to maintain it going forward and keep track of any issues that come in?

Reply:
Q:  Do you want to create a `patchers` directory in sdk_contrib for this, since all of the existing contributions are all middleware hooks?